src/Microsoft.Extensions.DependencyInjection/Resources.resx

```diff
@@ -137,4 +137,8 @@
   <data name="TypeCannotBeActivated" xml:space="preserve">
     <value>Cannot instantiate implementation type '{0}' for service type '{1}'.</value>
   </data>
+  <data name="NoConstructorMatch" xml:space="preserve">
+    <value>Unable to locate suitable constructor for type '{0}'. Ensure the type is concrete and services are registered for all parameters of a public constructor.</value>
```

Comment:
Change first sentence to: `A suitable constructor for type '{0}' couldn't be located.`

(All I did was translate from Robot-ese to Human-ese :smile: )